Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -947,4 +1011,179 @@ namespace AZ::Render
             }
         }
     }
+
+    // Find the trajectory path for a given actor instance
+    AtomActorDebugDraw::TrajectoryTracePath* AtomActorDebugDraw::FindTrajectoryPath(const EMotionFX::ActorInstance* actorInstance)
```

Comment:
suggestion: Could be worth mentioning in the API this is non-owning pointer that's returned (apologies if it already does)

Reply:
I will add the comment, but notice that it's part of the private function set so that might be a little over commenting.

Reply:
Sure, this is a preference thing but I find with larger functions some guide comments can be really helpful for other maintainers (this is one of my favourite articles about code comments that's worth a read http://antirez.com/news/124 🙂
)